### DISCUSSION TITLE
The HTTP QUERY Method

### DISCUSSION THREAD

**Discussion by skhaz** [fastapi/fastapi#12205] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
Will FastAPI support the HTTP QUERY method?

https://www.ietf.org/archive/id/draft-ietf-httpbis-safe-method-w-body-05.html
```

### Description

N/A

### Operating System

Other

### FastAPI Version

N/A

### Pydantic Version

N/A

### Python Version

3.12

**Comment by JavierSanchezCastro** [comment, marked as answer]
Related https://github.com/fastapi/fastapi/issues/5792
